install_files/securedrop-app-code/DEBIAN/postinst

```diff
@@ -24,8 +24,17 @@ case "$1" in
     chown www-data:www-data /var/www/document.wsgi
     chown www-data:www-data /var/www/source.wsgi
 
-    aa-complain /etc/apparmor.d/usr.sbin.tor
-    aa-complain /etc/apparmor.d/usr.sbin.apache2
+    a2dissite 000-default
```

Comment:
will add the comment. 

We also check for it in the ansible playbook but the issue was putting the securedrop apache apparmor profile in enforce mode as part of the securedrop-app-code deb package postinst. It would generate an apparmor error and break the install if the two default sites weren't disabled first since they aren't defined in the apparmor profile.